modular_skyrat/modules/New_drinks/code/modules/food_and_drinks/recipes/alcohol_reagents.dm

```diff
@@ -0,0 +1,23 @@
+/datum/reagent/consumable/ethanol/cum_in_a_hot_tub
+	name = "Cum in the Hot Tub"
+	boozepwr = 80
+	color = "#D2D7D9"
+	quality = DRINK_VERYGOOD
+	taste_description = "smooth cream"
+	glass_icon = 'modular_skyrat/modules/New_drinks/icons/obj/CIAH.dmi'
+	glass_icon_state = "cum_glass"
+	shot_glass_icon_state = "cum_shot"	//I'm funny, I know
+	glass_name = "Cum in the Hot Tub"
+	glass_desc = "Doesn't really leave it to the imagination, eh?"
+
+/datum/reagent/consumable/ethanol/cum_in_a_hot_tub_semen
+	name = "Cum in the Hot Tub"
+	boozepwr = 65
+	color = "#D2D7D9"
+	quality = DRINK_NICE
+	taste_description = "viscous cream"
+	glass_icon = 'modular_skyrat/modules/New_drinks/icons/obj/CIAH.dmi'
+	glass_icon_state = "cum_glass"
+	shot_glass_icon_state = "cum_shot"
+	glass_name = "Cum in the Hot Tub"
+	glass_desc = "The name is probably exactly what it is."
```

Comment:
Setting aside the fact that I'm personally not a fan of this, the two versions should have different names to avoid pref break.

Also, in general, the drink itself probably shouldn't have _that_ word in its name.

Reply:
Its a real drink and thats what its actually called.

Reply:
That doesn't make a difference for the first part of the comment, they should be different as to avoid pref-break.